src/core/plugins/deep-linking/manipHtmlStr.js

```diff
@@ -0,0 +1,31 @@
+import $ from "jquery"
```

Comment:
Let's use `jQuery` instead of `$` - I mostly see the `$` notation used when jQuery is in the global scope.

If Cheerio is used here, `cheerio` works as well.